source/dlangbot/github.d

```diff
@@ -199,8 +200,17 @@ Json[] tryMerge(in ref PullRequest pr, MergeMethod method)
         return commits;
     }
 
+    auto labelName = method.labelName;
+    auto events = ghGetRequest(pr.eventsURL).readJson[]
+        .retro
+        .filter!(e => e["event"] == "labeled" && e["label"]["name"] == labelName);
```

Comment:
An alternative would be to cache the event in Redis/Mongo as we most likely have already receive it. 
AFAIK with a limited amount of data, using both is free in Heroku.